tests/MongoDB.Driver.Examples/CausalConsistencyExamples.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Collections.Generic;
+using FluentAssertions;
+using MongoDB.Bson;
+using MongoDB.Driver.Core.TestHelpers.XunitExtensions;
+using Xunit;
+
+namespace MongoDB.Driver.Examples
+{
+    public class CausalConsistencyExamples
+    {
+        [Fact]
+        public void Causal_Consistency_Example_2()
+        {
+            RequireServer.Check().SupportsCausalConsistency();
+
+            GetClient().GetDatabase("myDatabase").DropCollection("myCollection");
+
+            // Start Tunable consistency controls example
+            var connectionString = "mongodb://localhost/?readPreference=secondaryPreferred";
```

Comment:
Do we run this test on Evergreen? If so what should the connection string be when running on Evergreen? Probably not this.

p.s. the current versions (on master) of evergreen.yml and build.cake do NOT run the examples on Evergreen.

Should they? (if so that should probably be a separate ticket)

Reply:
> Do we run this test on Evergreen?
Nope, but I'm planning to do this.

Reply:
Let's not worry about Evergreen right now.

We have never run these Documentation examples in Evergreen before.